flake.nix

```
@@ -135,6 +135,8 @@
         if nixosCfg.config.garnix.server.persistence.enable
         then nixosCfg.config.garnix.server.persistence.name + ".persistent.garnix.me"
         else getHash nixosCfg;
+
+      mkModules = mkModulesOpts: import ./mk-modules.nix { inherit flakeInputs mkModulesOpts; };
```

```suggestion
      mkModules = mkModulesOpts: import ./mk-modules.nix { inherit flakeInputs; } mkModulesOpts;
```

Maybe this is nicer, so that one parameter is imports (so to speak) and the other is a real function argument? Maybe `mk-modules.nix` would be easier to read? Very much TIOLI.

---

Yeah I went back and forth on this. I'll go with this (and also if we're doing this then maybe there's no reason to wrap `flakeInputs` in an attrset)